### ISSUE TITLE
about-to-send email getting incorrectly sorted below all published emails

### ISSUE THREAD

**Issue by james-baker** [buttondown-email/roadmap#1843]
a.k.a. the "dude, where's the email I just wrote!?" bug
(I suspect without any evidence that this is probably a null-sorting default-behavior problem as an about-to-send email probably has a null/undefined "published datetime")

Repro:
Write an email and send it.
Click out of the "about to send" dialog that pops up after the are-you-sure-yes.
Click on "Emails" to navigate to that page.

Expected: any emails page sorted by -published should have my latest email at the very top
Actual: the email I just wrote isn't even loaded on the page, though quick "Load more" clicks can eventually get to it at the very bottom of the list.

![image](https://github.com/buttondown-email/roadmap/assets/89615/2177fed9-a937-4720-9325-ba358ea37548)
Image: 108 sent and 1 about-to-send email, with only "Sent" stuff at the top of the list.
Behavior is the same with all-unchecked (default fresh page) or all-checked filters.

**Comment by jmduke** [comment]
@james-baker Yup, your understanding is correct; I'd basically (as you can see from the above labeling) argue that this is not a bug per ce but is still definitely not a good experience. Let me think a little bit about the right way to handle this; I think there's probably nothing wrong with just adding the publish date to an email when we cast it to `about to send`, which would fix this.

**Comment by james-baker** [comment]
I just realized all of the above applies to drafts too. The default "sort by -published" translates in my mind to "most recent stuff at the top of the list" and then it treats drafts as if they are the "oldest" thing.

**Comment by james-baker** [comment]
My experience lately is that this one has been fixed, is that true?
